clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -51,26 +59,42 @@ export async function loadProcessorGroupCopybookPaths(
 export async function loadProcessorGroupCopybookPathsConfig(
   item: { scopeUri: string },
   configObject: string[],
-): Promise<string[]> {
+  dialect?: string,
+): Promise<
+  (string | ZoweDatasetConfigModel | ZoweUssConfigModel | EndevorConfigModel)[]
+> {
   const cfg = await loadProcessorGroupSettings(
     item.scopeUri,
     "libs",
     [] as string[],
+    dialect,
   );
+  const configs: (
+    | string
+    | ZoweDatasetConfigModel
+    | ZoweUssConfigModel
+    | EndevorConfigModel
+  )[] = [];
+  const remotes = cfg.filter((cfg) => typeof cfg == "object");
+  const locals = cfg.filter((cfg) => typeof cfg === "string");
+
   const config = SettingsService.evaluateVariables(
-    [...cfg, ...configObject],
+    [...locals, ...configObject],
     getVariablesFromUri(item.scopeUri, false),
   );
 
   const wsUri = workspace.getWorkspaceFolder(Uri.parse(item.scopeUri))?.uri;
   if (wsUri === undefined) {
-    return configObject;
+    configs.push(...configObject);
+  } else {
+    const globs = globSync(
+      config.map((ele) => ele.replace(backwardSlashRegex, "/")),
+      { cwd: cleanWorkspaceFolderName(wsUri.fsPath), absolute: true },
+    ).map((s) => normalizePath(s));
+    configs.push(...globs);
   }
-  const globs = globSync(
-    config.map((ele) => ele.replace(backwardSlashRegex, "/")),
-    { cwd: cleanWorkspaceFolderName(wsUri.fsPath), absolute: true },
-  ).map((s) => normalizePath(s));
-  return globs;
+  configs.push(...remotes);
```

Comment:
The configuration order was lost here.

Reply:
shouldn't be locals then remotes ?

Reply:
The order specified by the user in the configuration should be respected.